### PR TITLE
Extract Stop words

### PR DESCRIPTION
**Pull request by navinrathore**
* Extracting stop words for each DONOT_USE field.
* stopWords are saved into csv files and html docs. The html docs have been linked in base documentation model.html
* Added new config param - **stopWordsCutoff** (default value = 0.1)

### CODE REVIEW

**Comment by navinrathore** [comment]
#78

File: client/src/main/java/zingg/client/Arguments.java [hunk 1]
```diff
@@ -507,7 +508,7 @@ public String getZinggBaseModelDir() {
 
 	@JsonIgnore
 	public String getZinggDocDir() {
-		return getZinggBaseModelDir();
+		return zinggDir + "/" + modelId;
```
Comment (sonalgoyal) [inline client/src/main/java/zingg/client/Arguments.java:511]: why do we need this change?

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

File: client/src/main/java/zingg/client/Arguments.java (hunk 1 above)
Comment (navinrathore) [reply client/src/main/java/zingg/client/Arguments.java:511]: getZinggDocDir() was unused.
getZinggBaseModelDir() is models/<id>/model directory. it contains model detail. 
The getZinggDocDir() seems corresponding to doc/documentation, therefore used after modification.

Comment (sonalgoyal) [reply client/src/main/java/zingg/client/Arguments.java:511]: thanks. so cant we then keep returning the getzinggBaseModelDir which is anyways what you are returning?

**Pull request merged**